.gitignore

```
@@ -25,3 +25,6 @@ target/
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+/node_modules/
```

:wrench: We may want to add a newline above the /node_modules entry so it's separated from the "virtual machine crash logs" block.
```suggestion

/node_modules/
```